src/components/ModelTag/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { ModelTag as DefaultModelTag, ModelIcon, ModelTagProps } from '@lobehub/icons';
+import { Tag } from '@lobehub/ui';
+import { memo } from 'react';
+
+import { featureFlagsSelectors, useServerConfigStore } from '@/store/serverConfig';
+import { useUserStore } from '@/store/user';
+import { modelProviderSelectors } from '@/store/user/selectors';
+
+const CustomModelTag = memo<ModelTagProps>(({ type = 'mono', model, ...rest }) => {
+  const enabledList = useUserStore(modelProviderSelectors.modelProviderListForModelSelect);
+
+  let modelCard;
+
+  for (const provider of enabledList) {
```

Comment:
If you want to display the model name, it's better to have a `provider` props to know the enabled provider. This lookup way will have confilct as the order.

Maybe a optional props with provider,if have this value to just get the model card, or use this lookup way.